code/controllers/configuration/entries/general.dm

```diff
@@ -361,7 +363,7 @@
 
 /datum/config_entry/flag/announce_admin_login
 
-/datum/config_entry/flag/allow_map_voting

```

Comment:
Leave this one in and set deprecated_by and update value so there's no need for change in config.

Reply:
all right